week-4/Homework/mandatory/3-water-bottle.js

```diff
@@ -50,7 +60,10 @@ Extra question:
   Leave your answer below:
 */
 
-// HERE COMES YOUR ANSWER
+/* HERE COMES YOUR ANSWER
+I think, using this helps us not writing the variable name repeatedly which keeps our code simpler and clean.
+Also if we did not use 'this' and if we need to use 'bottle' variable in an anoter object later on, 
+we would need to change so many lines of codes.*/

```

Comment:
Very good answer
`this` provides encapsulation which means you're calling only the items in the object scope (between the {}) and you're ensuring you are not affecting any other objects